### ISSUE TITLE
Enable custom target for added entry

### ISSUE THREAD

**Issue by daFish** [ninsuo/symfony-collection#56]
Hi there,

I'd like to propose a new configuration option to specify the target of the collection entry.

# Problem
I have customized the rendering of the entry and widget to be able to display the collection in a table. When I activate the plugin and adding a new entry, the entry isn't added as a new row to the table.

# Solution
Add a new option to the plugin to specify the target for the adding operation. The passed value is the selector or even an element:
```js
$('.my-selector').collections({
    prefix: 'test',
    target: '.table tbody'
});
```
```js
$('.my-selector').collections({
    prefix: 'test',
    children: [{
        target: '.table tbody'
    }]
});
```

This way the plugin would also be handy for special cases when dealing with collections.

**Comment by ninsuo** [comment]
Related issue:

Children of a collection are usually just below the collection itself, like:

```html
<div id="collection">
   <div id="child_0">(...)</div>
   <div id="child_1">(...)</div>
   <div id="child_2">(...)</div>
</div>
```

But when setting `elements_selector` option to something deeper than 1, like `elements_selector: "tbody > tr"` with collection in a `table`, the "add" button located at the bottom (`add_at_the_end` option set to true or all items removed from the collection) will add elements below the collection container (`<table>` in this case) instead of below `<tbody>`.

It will not be possible to programmatically fix this because collections may be created with 0 elements, thus plugin can't guess the right deepness using one existing item.

We should add a `elements_parent_selector` (yeah, naming is quite boring but this is to match with the current `elements_selector` option) that will be by default the collection itself, but that can be overwritten for more flexibility.

**Comment by daFish** [comment]
@ninsuo Your proposal about the new option seems plausible.

**Comment by ninsuo** [comment]
Implemented in #77 
Demo at: https://symfony-collection.fuz.org/symfony3/basic/inATable

Enjoy!
